.github/workflows/build_n_deploy_dev.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
 
   bygg:
     name: Bygg app/image og push til GitHub
```

Comment:
Både Bygg app/image og push til GitHub og Deploy til dev gcp er Status checks that are required i branch protection rule, så enten må vel disse fjernes eller så må man legge  if: ${{ github.actor != 'dependabot[bot]' }} på de stepene man ikke ønsker å kjøre som dependabot

Reply:
Fikset 👍

Reply:
@daphneleebeek Tror ikke det fikser det. 
https://github.com/navikt/familie-ks-soknad/settings/branch_protection_rules/27670543
Der står hvilke jobber som må ha kjørt for å få merget.

Hadde vært nok å sette
- uses: nais/docker-build-push@v0
  id: docker-push
  if: github.triggering_actor != 'dependabot[bot]'
  
  og samme på nais/deploy/actions/deploy@v1 step